docs/source/guides/jsonschema.rst

```diff
@@ -201,6 +201,7 @@ When naming columns in a "master table", we refer to the unique id of each row a
         "required": ["shifts", "resources_unavailable"]
     }
 
+As explained in the section beforehand, the parameters that are unidimensional should be on a table called ``parameters``.
 
 Example with TSP
 -------------------
```

Comment:
I do not understand why it is complaining here. In the github viewer it can be seen properly and on pycharm the same